README.md

```diff
@@ -1,67 +1,103 @@
 # Women Coding Community Platform Backend Service
 
-## Locally Development
+<!-- TOC -->
 
-* Build and run tests
+* [Women Coding Community Platform Backend Service](#women-coding-community-platform-backend-service)
+    * [Setup](#setup)
+        * [JAVA 21 with SDKMAN](#java-21-with-sdkman)
+        * [Setup IntelliJ](#setup-intellij)
+    * [Run Locally](#run-locally)
+    * [Open API Documentation](#open-api-documentation)
 
-```shell
-./gradlew clean build
-```
+<!-- TOC -->
 
-* Create Jar
-
-```shell
-./gradlew clean bootJar
-```
+## Setup
 
-* Start Spring Boot Application:
+### JAVA 21 with SDKMAN
```

Comment:
Good to include gradle version used for the project

Reply:
But that is not something you have to install, it is included into the wrapper itself. I don't think we should include here.